documentation/Installation.md

```diff
@@ -589,17 +590,17 @@ The following options are supported:
 
 |Name|Type|Mandatory|Default Value|Example|Description|
 |---|---|---|---|---|---|
-|keyfile|string|no| |`/home/username/.ssh/id_rsa`|**Absolute** path to keyfile on local machine to access the cluster machines, Either a keyfile or a password should be provided|
-|password|string|no| |`password@123`|Password to access the cluster machines, Either a keyfile or a password should be provided|
-|username|string|no|`root`|`centos`|Username for SSH-access the cluster machines|
-|name|string|no| |`k8s-control-plane-1`|Cluster member name. If omitted, Kubemarine calculates the name by the member role and position in the inventory. Note that this leads to undefined behavior when adding or removing nodes.|
-|address|ip address|no| |`10.101.0.1`|External node's IP-address|
-|internal_address|ip address|**yes**| |`192.168.0.1`|Internal node's IP-address|
-|connection_port|int|no|`22`| |Port for SSH-connection to cluster node|
-|connection_timeout|int|no|10|`60`|Timeout for SSH-connection to cluster node|
-|roles|list|**yes**| |`["control-plane"]`|Cluster member role. It can be `balancer`, `worker`, or `control-plane`.|
-|labels|map|no| |`netcracker-infra: infra`|Additional labels for node|
-|taints|list|no| |See examples below|Additional taints for node. **Caution**: Use at your own risk. It can cause unexpected behavior. No support is provided for consequences.|
+|keyfile|string|no| |`/home/username/.ssh/id_rsa`|The **Absolute** path to the keyfile on the local machine to access cluster machines. Either a keyfile or a password should be provided.|
```

Comment:
Thank you @shmo1218 for extra fixes of the documentation. Could you also fix corresponding sentences in `kubemarine/resources/schemas`? They repeat the sentences from Installation.md a lot.

Reply:
Hi, @ilia1243,

Sure. There is still some content that needs to be checked in installation.md. As soon as it is done, I will check the schemas.

Reply:
Hi, @ilia1243,

All schema files in the "resources" folder are reviewed. 

Thanks!

Reply:
Thank you @shmo1218. I see very many changes in the documentation that does not relate to the target of this PR: "Fixes and improvements of Kubernetes audit policy maintenance". I moved them to the separate branch https://github.com/Netcracker/KubeMarine/tree/documentation/installation_and_schema_review Please create new PR from it. Please also pay note that after commit https://github.com/Netcracker/KubeMarine/commit/c2417b2b45ff7efdfb1bd927270ff36e9f0744ed CI workflows was broken.

Reply:
Hi, @ilia1243,

I have created a new PR for the additional doc changes. Please take a look and do the needful.
https://github.com/Netcracker/KubeMarine/pull/498

Thanks!